app/controllers/crate/version.js

```diff
@@ -179,7 +178,14 @@ export default Controller.extend({
     },
 
     report: observer('crate.readme', function() {
-        setTimeout(() => $(window).trigger('hashchange'));
-    }),
-
+        if (typeof FastBoot === 'undefined') {
```

Comment:
This is probably a silly question, but I don't do much JS and I'm curious... Why is this better than `FastBoot === undefined`)? (I figure we can reasonably assume that nothing has redefined `undefined` in the crates.io code)

Reply:
@sgrif see https://stackoverflow.com/a/4725697/1478093